src/components/Table/types.d.ts

```diff
@@ -7,5 +7,6 @@ export interface TableColumn {
   order: boolean
   static?: boolean
   style?: any
+  formatTypeSort?: (value: any) => any
```

Comment:
What type of value can be actually passed here? We typically display text and numbers in tables.